### PR TITLE
ASimAuthenticationCiscoMeraki.yaml-9

### PR DESCRIPTION
**Pull request by Alekhya0824**
Required items, please complete
   
  Change(s):
   - Created a new file for Syslog and added Syslog parser in it. In Cisco Meraki file removed Syslog parser and in ASimAuthentication added Syslog Parser name  and same changes have done for vim as well.

   Reason for Change(s):
   - To Split the Syslog and Cisco Meraki parser separately

   Version Updated:
   - yes

   Testing Completed:
   - yes

   Checked that the validations are passing and have addressed any issues that are present:
   - yes

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Validation failures please check. Thanks
image

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please investigate the validation failures. Thanks

File: Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml:1]: why KQL tests changes showing?

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: ASIM Authentication parser for Cisco Meraki
-  Version: '0.1.1'
-  LastUpdated: Mar 18 2024
+  Version: '0.2.1'
+  LastUpdated: June 7 2024
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ References:
 - Title: Cisco Meraki Documentation
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml:15]: asim url

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: ASIM Authentication parser for Cisco Meraki
-  Version: '0.1.2'
-  LastUpdated: Apr 09 2024
+  Version: '0.2.1'
+  LastUpdated: June 7 2024
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml [hunk 5]
```diff
@@ -15,7 +15,7 @@ References:
 - Title: Cisco Meraki Documentation
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml:15]: asim url

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml [hunk 6]
```diff
@@ -0,0 +1,155 @@
+Parser:
+  Title: ASIM Authentication parser for Cisco Meraki
+  Version: '0.2.1'
+  LastUpdated: June 7 2024
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+Product:
+  Name: Cisco Meraki
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml:14]: url correction

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMerakiSyslog.yaml [hunk 8]
```diff
@@ -0,0 +1,232 @@
+Parser:
+  Title: ASIM Authentication parser for Cisco Meraki
+  Version: '0.2.1'
+  LastUpdated: June 7 2024
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMerakiSyslog.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMerakiSyslog.yaml [hunk 9]
```diff
@@ -0,0 +1,232 @@
...
+Product:
+  Name: Cisco Meraki
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMerakiSyslog.yaml:14]: url correction

**Review by vakohl — CHANGES_REQUESTED** [review]

File: Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml (hunk 1 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml:1]: I didn't get

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml (hunk 2 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml]: Done

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml (hunk 3 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml:15]: Done

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml (hunk 4 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml]: Done

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml (hunk 5 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml:15]: Done

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml (hunk 6 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml]: Done

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml (hunk 7 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml]: Done

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml (hunk 2 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml]: Done

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml (hunk 3 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml:15]: Done

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml (hunk 4 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml]: Done

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml (hunk 5 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml:15]: Done

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml (hunk 6 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml]: Done

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml (hunk 7 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMerakiSyslog.yaml]: Done

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMerakiSyslog.yaml (hunk 8 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMerakiSyslog.yaml]: Done

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMerakiSyslog.yaml (hunk 9 above)
Comment (Alekhya0824) [reply Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMerakiSyslog.yaml]: Done

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
One suggestion: once you're done with the open comment change, please click on the `Resolve Conversation` button. Thanks

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please check for failed validations. Thanks

File: Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml (hunk 1 above)
Comment (vakohl) [reply Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml:1]: image

**Review by vakohl — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**